neural_speed/convert/convert_bloom.py

```diff
@@ -96,6 +96,8 @@ def main(args_in: Optional[List[str]] = None) -> None:
     fout.write(struct.pack("i", 0))  # word_embed_proj_dim (for opt)
     fout.write(struct.pack("i", 0))  # do_layer_norm_before (for opt)
 
+    fout.write(struct.pack("i", 0))
+    fout.write(struct.pack("i", 0))
```

Comment:
a lot zeros, I know they are placeholders, can you give some comments that what are they?

Reply:
No problem